src/sdkUnitTests/src/main.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2020 Mnemosyne
+
+#include "function.h"
+
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "src/gtest-all.cc"
+#include "src/gmock-all.cc"
```

Comment:
Include what use use ;> gmock is not required at the moment

Reply:
done